### PR TITLE
01 test

### PR DESCRIPTION
**Pull request by poludnev**

### CODE REVIEW

File: config.env [hunk 1]
```diff
@@ -0,0 +1 @@
+NFT_STORAGE_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweDY3Y2NmMWU0OEU4NWViNzVFQzUzRmEzODU2NzZGOEVEM0Q2OWYxOWMiLCJpc3MiOiJuZnQtc3RvcmFnZSIsImlhdCI6MTY1NzcxMDM1NDA0OSwibmFtZSI6Im15a2V5In0.uem5KZz7-dLM4cgMoYgiBj9wwn4RV7hvhbvAtmrriQI
```
Comment (utgarda) [inline config.env:1]: pls put to github secrets

**Review by utgarda — CHANGES_REQUESTED** [review]
Doesn't work just yet

```
    /tmp/external-adapter    01-test ?1    npm run dev:ts               ✔ 

> nft.storage.proxy.client@1.0.0 dev:ts
> tsc && node dist/index.js

server started on port 9000
error Error: ENOENT: no such file or directory, open 'https:--twitter.com-ChainHackerClan-status-1616486060430589956.png'
    at Object.openSync (node:fs:590:3)
    at Object.readFileSync (node:fs:458:35)
    at Function.images.loadFromFile (/tmp/external-adapter/node_modules/images/index.js:213:37)
    at images (/tmp/external-adapter/node_modules/images/index.js:179:24)
    at getStampedImage (/tmp/external-adapter/dist/controllers/index.js:102:44)
    at Layer.handle [as handle_request] (/tmp/external-adapter/node_modules/express/lib/router/layer.js:95:5)
    at next (/tmp/external-adapter/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/tmp/external-adapter/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/tmp/external-adapter/node_modules/express/lib/router/layer.js:95:5)
    at /tmp/external-adapter/node_modules/express/lib/router/index.js:284:15 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'https:--twitter.com-ChainHackerClan-status-1616486060430589956.png'
}
node: symbol lookup error: /tmp/external-adapter/node_modules/images/vendor/linux-x64-binding.node: undefined symbol: _ZN2v87Isolate19CheckMemoryPressureEv
```

**Comment by Elizavetta** [comment]
>  fs.readFile("public/index.html", "utf-8")
    .then((data) => {
      response.set("Content-Type", "text/html");
      return response.status(200).send(data);
    })
    .catch((error) => {
      response.status(502).send(error);
    });
response.sendFile проще !?


> fetch("/send", {
        method: "POST",
        headers: { "Content-Type": "application/json" },
        body: JSON.stringify({ url: event.target.url.value }),
      })
url не проходит обработок


> const browser = await puppeteer.launch();
нет закрытия браузера ниже

не обработаны все промисы

File: config.env (hunk 1 above)
Comment (poludnev) [reply config.env:1]: Can't put to gh secrets, have no access to project settings.

**Comment by poludnev** [comment]
> 

fixed comments from the list

**Pull request merged**